### DISCUSSION TITLE
Using custom robots.txt

### DISCUSSION THREAD

**Discussion by AnastasiaTWW** [mkdocs/mkdocs#2822] [Q&A]
Hi all! If placing the robots.txt with custom rules to the docs_dir folder, will this custom robots.txt file fully re-write the automatically generated one? Or it will re-write only certain rule in robots.txt?

**Comment by fu-sen** [comment]
MkDocs does not automatically generate `robots.txt`.

If you generate `robots.txt` in the **docs/** folder, it will be built directly in the **site/** folder. You can expect the crawler to be referenced as is.
